vita3k/gui/src/settings_dialog.cpp

```diff
@@ -112,21 +98,23 @@ void draw_settings_dialog(GuiState &gui, HostState &host) {
     // Core
     if (ImGui::BeginTabItem("Core")) {
         ImGui::PopStyleColor();
-        if (fs::exists(fs::path(host.pref_path) / "vs0/sys/external")) {
-            ImGui::Text("Module List");
+        if (!gui.modules_list.empty()) {
+            ImGui::TextColored(GUI_COLOR_TEXT, "Module List");
             if (ImGui::IsItemHovered())
                 ImGui::SetTooltip("Select your desired modules.");
             ImGui::PushItemWidth(240);
-            static bool module_select[MODULES_COUNT] = { false };
-            if (ImGui::ListBoxHeader("##modules_list", MODULES_COUNT, 6)) {
-                for (int m = 0; m < MODULES_COUNT; m++) {
-                    auto modules = std::find(host.cfg.lle_modules.begin(), host.cfg.lle_modules.end(), LIST_MODULES[m]);
+            static char *modules_list = new char[gui.modules_list.size()];
+            static bool* module_select = new bool[gui.modules_list.size()]{ false };
```

Comment:
Ditto.
```suggestion
            static std::vector<bool> module_select;
```

Reply:
Same elad have send me this for bool can using  new char*